templates/partials/textbox.html

```diff
@@ -0,0 +1,54 @@
+{% set input_textbox_type = textbox_type %}
+
+{% if textbox_type == "password_input_text" %}
+    {% set input_textbox_id = "userPassword" %}
+{% else %}
+    {% set input_textbox_id = "userName" %}
+{% endif %}
+
+{% set textbox = {
+        "reg_input_text": {
+            "textbox_label": "Email or Username",
+            "textbox_name": "username",
+            "class": "input-textbox",
+            "input_type": "type = text",
+            "eye_icon_visible": "visibility-none-password-eye",
+            "more_info_title": "",
+            "more_info_text": "Enter Username or Email registered with the node you wish to access."
+        },
+        "password_input_text": {
+            "textbox_label": "Password",
+            "textbox_name": "userpassword",
+            "class": "input-textbox",
+            "input_type": "type = password",
```

Comment:
There was an issue with the quotes.  If the value was just "password" or "text" it would render with extra quotes and look something like '"password"', and it wouldn't work properly.